CONTRIBUTING.md

```diff
@@ -31,20 +66,35 @@ The workflow for doing so takes the following form:
 
 __Note: Before opening a pull request you must build your changes locally using Jekyll (see [README.md](README.md)) to verify that your changes build correctly and render as you expect.__
 
-Your pull request will be reviewed by other members of the community who may request changes.
-
 __Note: You can continue to push changes to your fork branch after you open a pull request  - the pull request will update accordingly__
 
+Your pull request will be reviewed by other members of the community who may request changes.
+Github provides an easy interface on it's website to apply (or reject) any reviewer-suggested changes with a click of a button.
+This avoids having to manually copy suggestions to your local copy and push back again.
+If you use the "Commit suggestion" button, you will need to update the local copy on your computer using `git pull` if you
+intend to push more edits from your computer.
+
 Once your pull request is approved, usually by at least two other community members, it will be merged into the fortran-lang.github.io master branch by the maintainers at which point it will be published to the fortran-lang.org site.
 
 If required, the repository maintainers can build a public preview of your proposed changes which will be available to view at `fortran-lang.org/pr/<pr_id>/` where `<pr_id>` is the numeric identifier of your pull request.
 
 This allows reviewers to directly view the generated result of your PR.
 After a pull request has been merged and successfully rendered, the maintainers will delete the preview build.
 
+__Note:__ if your pull request preview link doesn't work or doesn't update after re-building, then try adding a random parameter to the end of the URL, _e.g._ `https://fortran-lang.org/pr/98?v=2` - the name and value of the parameter don't matter, but use different values for each update.
+This will force the github content delivery network to serve you an updated version instead of a cached version which is out-of-date.
```

Comment:
```suggestion
This will force the GitHub content delivery network to serve you an updated version instead of a cached version which is out-of-date.
```